src/routes/bounce_details_page.jsx

```diff
@@ -42,6 +42,32 @@ export default class BounceDetailsPage extends React.Component {
 
   async componentDidMount() {
     const { match } = this.props;
+    const ws = new WebSocket(`ws://${process.env.SOCKET_URL}/ws`);
+
+    ws.onopen = () => {
+      ws.send(`check:${match.params.bounceRuleId}`);
+    };
+
+    ws.onmessage = msg => {
+      this.setState({
+        editText: msg.data,
+      });
+    };
+
+    ws.onclose = () => {
+      const { editText } = this.state;
+      if (editText === "EDIT" || editText === "ALREADY") {
```

Comment:
what does it mean when `editText` is `ALREADY`?

Reply:
we can store `editText === "EDIT" || editText === "ALREADY"` in a variable since it's being reused

Reply:
You guys are definitely right, "ALREADY" was pretty redundant